packages/wonder-blocks-tooltip/src/components/tooltip-popper.tsx

```diff
@@ -172,21 +217,43 @@ export default class TooltipPopper extends React.Component<Props> {
     }
 
     render(): React.ReactNode {
-        const {anchorElement, placement} = this.props;
+        const {anchorElement, placement, rootBoundary} = this.props;
+
+        const smallViewportModifier: SmallViewportModifier = {
+            name: "smallViewport",
+            enabled: true,
+            phase: "main",
+            fn: _modifyPosition,
+        };
+
+        const modifiers: (
+            | Partial<PreventOverflowModifier>
```

Comment:
I'm confused by the use of `Partial`. Are we saying that only part of the `PreventOverflowModifier` will be used or are we trying to say that these modifiers are optional?

Like should it be:

``` TS
type Modifier = PreventOverflowModifier | FlipModifier | SmallViewportModifier;
const modifiers: Modifier[] = [smallViewportModifier];
```

Likely I'm just confused.